docs/AppFramework.md

```diff
@@ -208,7 +208,7 @@ spec:
         secretRef: s3-secret
 ```
 
-6. Apply the Custom Resource specification: `kubectl apply -f SearchHeadCluster.yaml`
+6. Apply the Custom Resource specification: ` -f SearchHeadCluster.yaml`
```

Comment:
should be `kubectl apply -n splunk-operator -f SearchHeadCluster.yaml`

Reply:
fixed